package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@meemoo/react-components",
-  "version": "2.10.2",
+  "version": "2.9.0",
```

Comment:
Latest version is 2.10.2
http://do-prd-mvn-01.do.viaa.be:8081/#browse/browse:npm-viaa:%40meemoo%2Freact-components